contracts/mock/ERC20Extended1.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "../libraries/ERC20.sol";
+import "../interfaces/IERC20Extended.sol";
+
+contract ERC20Extended1 is ERC20, IERC20Extended {
```

Comment:
Mmm suggest to have a more descriptive name for the contract, like `MockERC20WithFee`